docs/eBpfExtensions.md

```diff
@@ -144,9 +144,26 @@ This structure is used to specify the address at which the various helper functi
 * `helper_function_address`: Array of addresses (64-bit unsigned integer) for the helper functions. The addresses must be arranged in the array in the *same order* as the array of helper function prototypes denoted by the `helper_prototype` field in `ebpf_program_info_t` struct.  For the correct execution of eBPF programs, the helper function addresses cannot change while a loaded eBPF program is executing.
 
 ### 2.2 Program Information NPI Client Attach and Detach Callbacks
-The eBPF Execution Context registers a Program Information NPI client module with the NMR for every eBPF program that gets loaded. The Execution Context will use the program type GUID of the program as the NPI ID of the client module. And as a result, upon eBPF program load, the associated Program Information NPI client module will attach with the corresponding Program Information NPI provider module in the extension. The Program Information NPI does not have any client or provider dispatch tables. Neither does the client's `NpiSpecificCharacteristics` have any data. So, no special processing is required in the client attach and detach callback handler on the provider module. An extension must not unload until there are no more attached Program Information NPI clients.
+The eBPF Execution Context registers a Program Information NPI client module with the NMR for every eBPF program that gets loaded. The Execution Context will use the program type GUID of the program as the NPI ID of the client module. And as a result, upon eBPF program load, the associated Program Information NPI client module will attach with the corresponding Program Information NPI provider module in the extension. The Program Information NPI does not have any provider dispatch tables, but has a client dispatch table. Client's `NpiSpecificCharacteristics` does not have any data though. The provider must store the following in a per-client data structure from the passed in parameters:
```

Comment:
```suggestion
The eBPF Execution Context registers a Program Information NPI client module with the NMR for every eBPF program that gets loaded. The Execution Context will use the program type GUID of the program as the NPI ID of the client module. And as a result, upon eBPF program load, the associated Program Information NPI client module will attach with the corresponding Program Information NPI provider module in the extension. The Program Information NPI does not have any provider dispatch tables, but has a client dispatch table. The client's `NpiSpecificCharacteristics` does not have any data though. The provider must store the following in a per-client data structure from the passed in parameters:
```

Reply:
Separately. please break into multiple source lines to make github diffs easier.   (I'd say the https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md statement that

> Source lines MUST NOT exceed 120 columns.

applies to markdown files too.)

Reply:
Updated sections 2.2 and 2.3 to not exceed 120 columns.